### ISSUE TITLE
[feature request] output logs to stdout

### ISSUE THREAD

**Issue by yeya24** [dragonflyoss/Dragonfly#651]
## Why you need it?

Dragonfly is used in cloud native situations and its components like dfdaemon and supernode maybe run as containers.

But currently the detailed logs are stored in logs file inside the container instead of outputing to stdout. If I want to see the logs file, I have to exec into the container first. It is better to have an option to configure logs output to stdout so I can use `docker logs` or `kubectl logs` to see the logs.

## How it could be?

Add an option or make it default to output logs to stdout

**Comment by inoc603** [comment]
I agree on this. Logs for dfdaemon and supernode should be output to stdout by default. It's easier for debugging and developing, and we can utilize existing tools like `journalctl`, `docker logs` and `kubectl logs` to view the logs. Console hook is alraedy implemented in `dflog`, and used in `dfget`. It would be easy to enable them in dfdaemon and supernode.

Also I did some refactoring on logging a while back [here](https://github.com/inoc603/Dragonfly/tree/log) which aims to make initializing the logger easier and more consistent across the whole project.

**Comment by yeya24** [comment]
Hi @inoc603, can you open a pr for this feature?

**Comment by inoc603** [comment]
@yeya24 Sure. I'll open one this weekend.

**Comment by yeya24** [comment]
I am sorry that I reopen this issue. I found that there is still no option to configure outputting logs to console in both dfdaemon and supernode. Did I miss something? @Starnop  @lowzj  @inoc603

**Comment by lowzj** [comment]
@yeya24 please see [pkg/dflog/log.go#WithConsole](https://github.com/dragonflyoss/Dragonfly/blob/master/pkg/dflog/log.go#L115)

**Comment by stanxing** [comment]
> please see pkg/dflog/log.go#WithConsole

I also want to ask about the status of the issue. I deployed dragonfly(version 1.0.4) in k8s, where dfclient is deployed as daemonset and supernode as deployment. I hope to see the log from `kubectl logs -f`.

- dfclient generates three logs files, which are `dfdaemon.log`, `dfclient.log` and `dfserver.log`. 

The config  `logConfig.path:/dev/stdout` in `/etc/dragonfly/dfdaemon.yml` is only effective for `dfdaemon.log`. But the config `logConfig.path:/dev/stdout` in `/etc/dragonfly/dfget.yml` is invalid for  `dfclient.log` and `dfserver.log`.

- supernode generates `app.log` file.

`base.logConfig.path:/dev/stdout` config in `/etc/dragonfly/supernode.yml` is also invalid. Log will continue to wirte to app.log.

please correct me if it is wrong.
